src/discof/shredcap/fd_shredcap_tile.c

```diff
@@ -105,7 +137,28 @@ scratch_align( void ) {
 
 FD_FN_PURE static inline ulong
 loose_footprint( fd_topo_tile_t const * tile FD_PARAM_UNUSED ) {
-  return 1UL * FD_SHMEM_GIGANTIC_PAGE_SZ;
+  /* manifest_bank_mem + manifest_spad_mem + others */
+  return 4UL * FD_SHMEM_GIGANTIC_PAGE_SZ;
+}
+
+FD_FN_CONST static inline ulong
+manifest_bank_align( void ) {
+  return 128UL;
+}
+
+FD_FN_CONST static inline ulong
+manifest_bank_footprint( void ) {
+  return 2UL*1024UL*1024UL*1024UL;
+}
+
+FD_FN_CONST static inline ulong
+manifest_spad_align( void ) {
+  return 128UL;
+}
+
+FD_FN_CONST static inline ulong
+manifest_spad_footprint( void ) {
+  return 1UL*1024UL*1024UL*1024UL;
```

Comment:
these magic values seem a little strange. are they just copied over from the snapshot tile?